wolfssl/wolfio.h

```diff
@@ -163,15 +161,9 @@
             #include <sys/socket.h>
             #include <arpa/inet.h>
             #include <netinet/in.h>
-            #ifdef HAVE_NETDB_H
```

Comment:
Why`HAVE_NETDB_H` and `HAVE_SYS_IOCTL_H` is removed? :cry:

Reply:
See #7381